com.unity.render-pipelines.high-definition/Runtime/Lighting/LightCookieManager.cs

```diff
@@ -181,8 +181,13 @@ Texture FilterAreaLightTexture(CommandBuffer cmd, Texture source, int finalWidth
                     m_MPBFilterAreaLights.SetInt(s_sourceMipLevel, 0);
                     m_MPBFilterAreaLights.SetTexture(s_texSource, source);
 
+
+                    // Since we blit the cookie texture into a common texture, to avoid leaking we blit with an extra border
+                    int border = 1;
                     cmd.SetRenderTarget(m_TempRenderTexture0, 0);
-                    cmd.SetViewport(new Rect(0, 0, viewportWidth, viewportHeight));
+                    cmd.SetViewport(new Rect(0, 0, viewportWidth + border, viewportHeight + border));
```

Comment:
wondering if it cause any atlas fitting issue? i.e you have a 256x256 atlas with a 256x256 texture and then it don't fit anymore because of the 1 border. Maybe we should "rescale" ? i.e add a border of 1 and render in 254x254 ? Thought

Reply:
So as it is now it will fit, it will just not put the border around it which is fine as there is no data outside to be messed with. 

This only changes the viewport, not the size of the texture, so in your example what would happen is the border will end up outside the rendertarget, but that should be fine as in that case we only have the one thing.